File: appkey-service/src/main/java/org/appkey/data/UserAppkey.java
package org.appkey.data;

import org.springframework.data.cassandra.core.mapping.PrimaryKey;
import org.springframework.data.cassandra.core.mapping.Table;
import lombok.*;

@Table("sm01")
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@EqualsAndHashCode
@ToString
public class UserAppkey {

    @PrimaryKey
    private String appkey;

    private String userid;
    private int usertype;


}

File: appkey-service/src/main/kotlin/org/appkey/data/UserAppkey.kt
package org.appkey.data

import lombok.*
import org.springframework.data.cassandra.core.mapping.PrimaryKey
import org.springframework.data.cassandra.core.mapping.Table

@Table("sm01")
@NoArgsConstructor
@AllArgsConstructor
@EqualsAndHashCode
@ToString
class UserAppkey {

    @PrimaryKey
    var appkey: String? = null
    var userid: String? = null
    var usertype = 0
}